quipucords/scanner/job.py

```
@@ -89,7 +90,8 @@ def run(self):
 
                 message = 'FATAL ERROR. %s' % str(error)
                 self.scan_job.fail(message)
-                raise error
+                task_status = ScanTask.FAILED
```

We want to re-raise the error here to get the stacktrace.